### DISCUSSION TITLE
Expandable FAB

### DISCUSSION THREAD

**Discussion by AbdullahAbuAjwa** [darioielardi/flutter_speed_dial#238] [Ideas]
Thank you for your amazing library.
Can you add this feature

![image](https://user-images.githubusercontent.com/46073236/156917929-7b87a5a2-55ff-4fe5-adf9-a573ac07f8e5.png)

**Comment by HassanGhazy** [comment]
I agree.

**Comment by xanaddams** [comment]
also on my req list
